librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -3,6 +3,248 @@
 ---
 name: cmd_debug
 commands:
+  - name: db
+    summary: Breakpoints commands
+    subcommands:
+      - name: db
+        summary: List breakpoints / Add breakpoint
+        cname: cmd_debug_db
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+           - RZ_OUTPUT_MODE_STANDARD
+           - RZ_OUTPUT_MODE_RIZIN
+           - RZ_OUTPUT_MODE_JSON
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
```

Comment:
Ok it's a pita that this one can't be removed, otherwise you have no difference between `db` and `db <addr>`. One solution could be to have `dbl` to list breakpoints, which would make sense actually... @XVilka WDYT?

Reply:
`dbl` is ok, that's what we did in some other cases, e.g. in `ax`  - we made a new command `axl` to list xrefs.